src/components/Editor/FreeBusy/FreeBusy.vue

```diff
@@ -169,9 +169,14 @@ export default {
 			const resources = []
 
 			for (const attendee of [this.organizer, ...this.attendees]) {
+				let title = attendee.commonName || attendee.uri.substr(7)
+				if (attendee === this.organizer) {
+					title += ` ${this.$t('calendar', '(organizer)')}`
```

Comment:
```suggestion
					title = ` ${this.$t('calendar', '{organizer} (organizer)', {organizer: title})}`
```
For correct i18n for languages in which the order of words might not be the same.

Reply:
Done